### PR TITLE
Split test file into per function files

### PR DESCRIPTION
**Pull request by ashleymichal**
a smol change to satisfy an itch. making separate from a test PR for SPAs

### CODE REVIEW

File: test/getAssetFromKV.js [hunk 1]
```diff
@@ -2,29 +2,8 @@ import test from 'ava'
...
-
-test('mapRequestToAsset() correctly changes /about.me/ -> /about.me/index.html', async t => {
-  let path = '/about.me/'
-  let request = new Request(`https://foo.com${path}`)
-  let newRequest = mapRequestToAsset(request)
-  t.is(newRequest.url, request.url + 'index.html')
-})
-
 test('getAssetFromKV return correct val from KV and default caching', async t => {
+  mockGlobal()
```
Comment (victoriabernard92) [inline test/getAssetFromKV.js]: why not setting global?

Comment (ashleymichal) [reply test/getAssetFromKV.js]: what do you mean?

Comment (victoriabernard92) [reply test/getAssetFromKV.js]: Curious why this line needed changing

Comment (ashleymichal) [reply test/getAssetFromKV.js]: i'm not sure? it is using `Request`, which is part of this global thing, and was apparently not being called before?

Comment (ashleymichal) [reply test/getAssetFromKV.js]: ooooo mockGlobal is called once per file it looks like.

**Pull request merged**